draft-ietf-ppm-dap.md

```diff
@@ -1239,30 +1247,27 @@ struct {
 } PrepareStep;
 
 struct {
+  u16 round;
```

Comment:
nit: since `AggregateInitializeResp` is only used for the first round, `round` will always be `0`. We could drop the field entirely from `AggregateInitializeResp` and save a few bytes.

Reply:
You're right. And on subsequent rounds, we require the helper to echo back the round value that the leader sent in its request, so even there it carries no information and is just an opportunity for the helper to have a bug. So it seems like we'd want to stop using `struct AggregationJob` in three places and go back to `AggregateContinueReq` and `AggregateContinueResp`. I was trying to do a REST thing here -- the servers send a REpresentation of the STate of the aggregation job back and forth -- but I'm backing away from that particular ledge now.

Reply:
I re-introduced `AggregationJobContinueReq` and `AggregationJobResp` messages. `AggregationJobContinueReq` contains the round field and is sent by the leader to helper.  `AggregationJobResp` omits the round field and is used as the response to both `AggregationJobInitReq` and `AggregationJobContinueReq`. I now wish I hadn't collapsed all the message types together back in #398, but so it goes.